test/js-native-api/test_string/test.js

```diff
@@ -18,6 +18,7 @@ assert.strictEqual(test_string.TestLatin1ExternalAutoLength(empty), empty);
 assert.strictEqual(test_string.TestUtf16ExternalAutoLength(empty), empty);
 assert.strictEqual(test_string.Utf16Length(empty), 0);
 assert.strictEqual(test_string.Utf8Length(empty), 0);
+assert.strictEqual(test_string.TestPropertyKeyUtf16(empty), empty);
```

Comment:
The function `TestPropertyKeyUtf16` has to be exposed to JS via declarations at https://github.com/nodejs/node/blob/main/test/js-native-api/test_string/test_string.c#L390.

Reply:
thank you very much for your suggestions, I wonder if this will be enough https://github.com/nodejs/node/pull/50282/commits/fcfa24c82fb955aeed40592aeaeeec2bc9db00cc @legendecas